src/main/java/de/zalando/ep/zalenium/container/kubernetes/KubernetesContainerClient.java

```diff
@@ -139,6 +140,14 @@ private void buildResourceMaps() {
                 }
             }
         }
+
+        String imagePullPolicyEnvValue = environment.getStringEnvVariable("ZALENIUM_KUBERNETES_IMAGE_PULL_POLICY", null);
```

Comment:
Maybe you could use `Optional.ofNullable` here, just to make it a bit more clean. [Here](https://github.com/zalando/zalenium/blob/master/src/main/java/de/zalando/ep/zalenium/proxy/DockeredSeleniumStarter.java#L160) is an example.

Reply:
Sorry about the whitespaces, something went funky when I switched from ubuntu to windows and copied the project -_- I'll fix it in the last commit.

Can you elaborate on the above - shouldn't it be `Optional.Of` for a String? Or perhaps, I am misunderstanding. Can you give an example of what you'd change it to?

Reply:
No worries! Thanks for fixing the formatting.

I think I just made it more complicated with my comment, reading it a second time, perhaps this is more simple:

```java
String imagePullPolicyEnvValue = environment.getStringEnvVariable("ZALENIUM_KUBERNETES_IMAGE_PULL_POLICY", ImagePullPolicyType.Always.name());
```

Since the `getStringEnvVariable` will return the second parameter when the first one is `null`. Then, the if-else is not needed. What do you think?